srcpkgs/faust/template

```diff
@@ -0,0 +1,51 @@
+# Template file for 'faust'
+pkgname=faust
+version=2.20.2
+revision=1
+build_style=gnu-makefile
+hostmakedepends="cmake pkg-config libmicrohttpd-devel llvm llvm-libunwind-devel"
+short_desc="Functional Programming Language for Real Time Signal Processing"
+maintainer="mag <mag-one@autistici.org>"
+license="GPL-2.0-or-later"
+homepage="http://www.faust.grame.fr"
+distfiles=https://github.com/grame-cncm/faust/releases/download/${version}/${pkgname}-${version}.tar.gz
+checksum="c32588ee60d0f7ca25034d3d5436def6b35495f4d3beb3b02208d48684bcfbe9"
+nostrip_files="libsndfile.so "
+noshlibprovides="libsndfile.so"
+
+case $XBPS_MACHINE in
```

Comment:
you may want to test for `$XBPS_TARGET_MACHINE` instead, and add libexecinfo-devel to  `makedepends=` (not host...)